### DISCUSSION TITLE
How do I finish the installation?

### DISCUSSION THREAD

**Discussion by charleslcso** [ghostfolio/ghostfolio#1102] [Q&A] [answered]
Hello,

Following the Self-hosting guide to install ghostfolio.

I downloaded the Docker image from Docker Hub, ran a. Run environment.

`docker ps` shows postgres and redis are both running.

Then what? There is nothing running in port 3333.

Kindly help.

**Comment by charleslcso** [comment, marked as answer]
Oh. Me bad. The Docker image assumes amd64, but I run it in arm64. May be it won't be supported.

**Reply by dtslvr** [reply]
Hello @charleslcso

I think your idea is related to https://github.com/ghostfolio/ghostfolio/discussions/1030.
